backend/src/impl/auth.py

```diff
@@ -103,9 +123,19 @@ def get_user() -> User:
 
 
 class CognitoClient:
-    # keys for fields in returned user_info
+    """
+    keys for fields in returned user_info
+    - sub: a unique identifier (UUID) for the authenticated user.
```

Comment:
The naming `sub` is a bit confusing. Since you explain it as a unique identifier (uuid) would `uuid` or `unique_id` be better?